src/Collapse/Collapse.component.tsx

```diff
@@ -108,32 +103,22 @@ export class Collapse extends React.Component<Props, any> {
 
   // Detect a new collapse state from props.isOpen change
   static getDerivedStateFromProps(props: Props, state: State) {
-    const isOpenProps =
-      props.isOpen !== undefined
-        ? !!props.isOpen && !props.isCollapsed
-        : !!props.isOpen || !props.isCollapsed;
-    const isOpen =
-      state.collapseState === EXPANDED || state.collapseState === EXPANDING;
-    const isCollapsed =
-      state.collapseState === COLLAPSED || state.collapseState === COLLAPSING;
-
-    const expand = !isOpenProps && isCollapsed;
-    const collapse = isOpenProps && isOpen;
+    const isOpenProps = props.isOpen === props.isOpen;
+    // const isOpen =
+    // state.collapseState === COLLAPSED || state.collapseState === COLLAPSING;
+
+    const expand = isOpenProps && !props.isOpen;
```

Comment:
I think this should just be `const expand = props.isOpen`, then get rid of the line 106 above

Reply:
Correct. Will fix it.